source/nanomsg/wrap.d

```diff
@@ -335,6 +335,14 @@ struct NanoSocket {
         return text(_protocol, connText, _uri);
     }
 
+    /// get the underlying nanomsg socket file descriptor.
+    /// Bear in mind that NanoSocket contains state about
+    /// the socket options, so changing these from the
+    /// raw fd will result in unpredictable behaviour
```

Comment:
Shouldn't it be `@system` then?

Reply:
I mean, maybe? `@safe` is really hard to reason about :(

Better (un)safe than sorry I guess, I'll change it now

Reply:
done